VideoQuickStart/ViewController.swift

```diff
@@ -201,8 +201,17 @@ class ViewController: UIViewController {
         let backCamera = CameraSource.captureDevice(position: .back)
 
         if (frontCamera != nil || backCamera != nil) {
+
+            let options = CameraSourceOptions { (builder) in
+                // Track UIWindowScene events. Conditionally compiled to support Xcode 10.x.
+                #if XCODE_1100
+                if #available(iOS 13.0, *) {
+                    builder.orientationTracker = UserInterfaceTracker(scene: self.view.window!.windowScene!)
+                }
+                #endif
```

Comment:
Needs to be done later, so that the window/scene is not nullable or by reaching up to the app delegate.

Reply:
Used the application, this should be sufficient since in our .plist multiple windows are not allowed.